server/src/__tests__/routes/comments/index.test.js

```diff
@@ -0,0 +1,79 @@
+import request from 'supertest';
+import application from '../../../configs/app';
+import Comments from '../../../models/comments';
+import Blogs from '../../../models/blogs';
+
+let server;
+const applicationInstance = application();
+
+const blogs = [{
```

Comment:
just create a file called test utils and move all this to over there, so you dont have to create the blogs/comments everytime.